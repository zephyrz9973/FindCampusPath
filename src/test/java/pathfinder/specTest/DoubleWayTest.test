#Test the behavior when there is a double way between two Nodes

CreateGraph graph3

AddNode graph3 A
AddNode graph3 B
AddNode graph3 C

AddEdge graph3 A B 5.0
AddEdge graph3 B A 5.0
AddEdge graph3 C B 6.0
AddEdge graph3 B C 6.0
AddEdge graph3 C A 23.0
AddEdge graph3 A C 23.0

FindPath graph3 A C

FindPath graph3 C A
