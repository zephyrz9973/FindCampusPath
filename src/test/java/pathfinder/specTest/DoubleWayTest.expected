#Test the behavior when there is a double way between two Nodes

created graph graph3

added node A to graph3
added node B to graph3
added node C to graph3

added edge 5.000 from A to B in graph3
added edge 5.000 from B to A in graph3
added edge 6.000 from C to B in graph3
added edge 6.000 from B to C in graph3
added edge 23.000 from C to A in graph3
added edge 23.000 from A to C in graph3

path from A to C:
A to B with weight 5.000
B to C with weight 6.000
total cost: 11.000

path from C to A:
C to B with weight 6.000
B to A with weight 5.000
total cost: 11.000
